### PR TITLE
added: auth

### PR DESCRIPTION
**Pull request by harshalini**
Added authentication in notes app
Now a user can Signup, login through the app with required credentials.
Used mockbee backend APIs for functionality

Preview -
https://beamish-bombolone-0bf080.netlify.app/

### CODE REVIEW

File: src/context/noteContext.js [hunk 1]
```diff
@@ -30,7 +31,6 @@ const NoteDataProvider = ({ children }) => {
   const AddNewNote = async (note) => {
     const getNote = await AddNoteHandler(note)
     setNote(getNote.data.notes)
-    console.log(note)
   }
```
Comment (MonikaJhingan) [inline src/context/noteContext.js:34]: Use of async and await has been nicely done.

File: src/noteServices/addNoteService.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+  const authToken = localStorage.getItem("authToken")
   try {
     const res = await axios.post(
       "/api/notes",
       {
         note: data 
       },
       {
-        headers: { authorization: token }
+        headers: { authorization: authToken }
```
Comment (MonikaJhingan) [inline src/noteServices/addNoteService.js:11]: Use of token for authentication is correctly  done.

**Review by MonikaJhingan — COMMENTED** [review]
The code looks well indented and proper modules are imported.

File: src/pages/auth/login.js [hunk 3]
```diff
@@ -1,22 +1,49 @@
...
-                            <button type="submit" className="account-btn"
+                            <button type="button" className="account-btn"
+                            onClick={(e) => {
+                                e.preventDefault();
+                                UserLoginHandler(userLogIn)
+                            }}
                             >Login</button>
                             <a href="#" className="fr-pswd">Forgot password?</a>
-                            <button type="submit" className="account-btn"
+                            <button type="submit" className="account-btn"
```
Comment (MonikaJhingan) [inline src/pages/auth/login.js:46]: The code is very well structured in this file .

File: src/styles/home.css [hunk 4]
```diff
@@ -66,7 +66,15 @@ nav ul li {
 }
 
 .login {
-    color: var(--black);
+    color: var(--white);
+}
```
Comment (MonikaJhingan) [inline src/styles/home.css:70]: Use of var for color representation is good. Use of rem unit is consistently done.

**Review by MonikaJhingan — APPROVED** [review]

**Review by Rohitprasad83 — APPROVED** [review]
Code looks good and readable

File: src/noteServices/addNoteService.js (hunk 2 above)
Comment (harshalini) [reply src/noteServices/addNoteService.js:11]: Thank you!

**Pull request merged**